chutney/action-impl/src/main/java/com/chutneytesting/action/kafka/KafkaBasicConsumeAction.java

```diff
@@ -62,6 +62,8 @@ public class KafkaBasicConsumeAction implements Action {
     static final String OUTPUT_BODY_PAYLOAD_KEY = "payload";
     static final String OUTPUT_HEADERS = "headers";
     static final String OUTPUT_PAYLOADS = "payloads";
+    static final String OUTPUT_BODY_KEYS_KEY = "key";
```

Comment:
static final String OUTPUT_BODY_KEY_KEY = "key"; to match the pattern used for 
static final String OUTPUT_BODY_PAYLOAD_KEY = "payload";